tests/conftest.py

```diff
@@ -89,7 +89,7 @@ def _parse_model():
 
 
 @pytest.fixture(scope='session')
-def create_model(load_example):
+def create_model_for_test_with_pheno_dataset(load_example_model_for_test):
```

Comment:
It looks really nice to me and I have no comment but a random thought, maybe create_model_for_test is enough? It sounds more general and maybe we will add more test datasets in the future?

Reply:
Done!